src/main/java/com/github/manolo8/darkbot/gui/login/LoginForm.java

```diff
@@ -100,7 +100,10 @@ protected LoginData doInBackground() {
                 LoginUtils.findPreloader(loginData);
                 return loginData;
             } catch (Exception e) {
-                publish(new Message(true, "Failed to login!", IssueHandler.createDescription(e)));
+                int i = e.getMessage().indexOf("-");
```

Comment:
Don't use a "-" and a substring. Change LoginException to have a different field for message or similar, and catch for LoginException separately of regular exception